feature/HW-17/index.js

```diff
@@ -0,0 +1,87 @@
+// let first = 'true';
+// let second = false;
+// let third = 17;
+// let fourth = undefined;
+// let fifth = null;
+// console.log(typeof(first), typeof(second), typeof(third), typeof(fourth), typeof(fifth));
+
+
+// let height = 15;
+// let width = 20;
+
+// if (height < width) {
+//     console.log(width)
+// }
+// else {
+//     console.log(height);
+// }
+
+
+// let b = 20;
+
+// for (let a = 1; a < b; a += 1) {
+//     if (a % 3 !== 0) {
+//         continue;
+//     }
+//     console.log(a);
+// }
+
+
+// let key = true;
+// let documents = true;
+// let pen = true;
+// let apple = false;
+// let orange = true;
+// let shoulGoToWork;
+// if ((key == true && documents == true && pen == true) && (apple == true || orange == true)) {
+//     shoulGoToWork = true;
+// } else {
+//     shoulGoToWork = false;
+// }
+// console.log(shoulGoToWork);
+
+
+// let n = +prompt();
+// if ((n % 3 != 0) && (n % 5 == 0)) {
+//     console.log('Fiz');
+// } else if ((n % 3 == 0) && (n % 5 != 0)) {
+//     console.log('Buz');
+// } else if ((n % 3 == 0) && (n % 5 == 0)) {
+//     console.log('FizBuz');
+// }
+
+
+
+// let age = +prompt('Возраст?');
+// if (18 <= age) {
```

Comment:
1. По-хорошему первой  должна идти переменная. Так семантически верно
2. В условии строго больше 18
```suggestion
// if (age > 18) {
```